client/src/app/page/wrapper/wrapper.component.ts

```diff
@@ -38,7 +41,41 @@ export class WrapperComponent implements OnInit {
 
   getTasks(): void {
     this.tasksService.getTasks()
-      .subscribe(tasks => this.tasks = tasks);
+      .subscribe(tasks => {
+        this.tasks = tasks.map( (item: any) => {
+          return {
+            id: item._id,
+            name: item.name,
+            excerpt: '',
+            status: {name: item.status, value: this.getStatusValue(item.status)},
+            type: {name: item.type, value: this.getTaskType(item.type)},
+            date: this.convertDate(item.date),
+            author: '',
+            content: item.content
+          };
+        });
+      });
+  }
+
+  getStatusValue = (status: string): number => {
```

Comment:
Looks like you should use enum like this:

enum Status {
    High = 0,
    Normal,
    ...
}